### PR TITLE
Adds HighCohesion details to Adopter doc

### PR DESCRIPTION
**Pull request by jgsamblas**
# Changes

- Adds HighCohesion details to Adopters document

/kind documentation

### CODE REVIEW

File: ADOPTERS.MD [hunk 1]
```diff
@@ -31,5 +31,6 @@ If you would like to be included in this table, please submit a comment to [this
 | VMware                             | Cloud Native Runtimes for VMware Tanzu                                 | Generally Available      | [link](https://network.tanzu.vmware.com/products/serverless)         |
 | VMware                             | VMware Event Broker Appliance                                          | Development              | [link](https://vmweventbroker.io/)                                   |
 | vorteil pty ltd*                   | Serverless orchestration engine                                        | Development & Production | [link](http://vorteil.io)                                            |
+| HighCohesion                       | SaaS E-commerce Integration Platform                                   | Development              | [link](https://www.highcohesion.com/)                                |
```
Comment (csantanapr) [inline ADOPTERS.MD:34]: Can you insert in alphabetically order? it would be after the row `IBM`

**Review by csantanapr — CHANGES_REQUESTED** [review]

File: ADOPTERS.MD [hunk 2]
```diff
@@ -31,5 +32,4 @@ If you would like to be included in this table, please submit a comment to [this
 | VMware                             | Cloud Native Runtimes for VMware Tanzu                                 | Generally Available      | [link](https://network.tanzu.vmware.com/products/serverless)         |
 | VMware                             | VMware Event Broker Appliance                                          | Development              | [link](https://vmweventbroker.io/)                                   |
 | vorteil pty ltd*                   | Serverless orchestration engine                                        | Development & Production | [link](http://vorteil.io)                                            |
-
-`*` organization that is an end user of Knative and not a vendor/reseller
\ No newline at end of file
+`*` organization that is an end user of Knative and not a vendor/reseller
```
Comment (csantanapr) [inline ADOPTERS.MD:35]:
```suggestion

`*` organization that is an end user of Knative and not a vendor/reseller
```

**Review by csantanapr — APPROVED** [review]

File: ADOPTERS.MD (hunk 1 above)
Comment (vaikas) [reply ADOPTERS.MD]: Well, just wait a minute, if we want it alphabetically (and we do), how does it come _after_ IBM @csantanapr ?

**Pull request merged**